src/Cake.Svn/SvnAliases.Info.cs

```diff
@@ -0,0 +1,255 @@
+        /// <code>
+        /// <![CDATA[
+        ///     var svnInfoSettings = new SvnInfoSettings
+        ///     {
+        ///         Depth = SvnDepth.Unknown;
+        ///     };
+        ///
+        ///     var localFileInfo = GetFileInfo(@"C:\project\src\", svnInfoSettings);
+        ///
+        ///     Verbose("Path: {0}", localFileInfo.Path);
+        /// ]]>
+        /// </code>
+        /// </example>
+        [CakeMethodAlias]
+        [CakeAliasCategory("Info")]
+        [CakeNamespaceImport("Cake.Svn.Info")]
+        public static SvnInfoResult GetFileInfo(this ICakeContext context, FilePath filePath, SvnInfoSettings settings)
+        {
+            var svnInformation = new SvnInfo(context.Environment, SvnClientFactoryMethod);
+
+            // A file returns only one SvnInfoResult.
+            return svnInformation.GetInfo(filePath, settings ?? new SvnInfoSettings()).Single();
+        }
+
+        /// <summary>
+        /// Gets Subversion information about the file or directory at <paramref name="fileOrDirectoryPath"/>.
+        /// </summary>
+        /// <param name="context">The Cake context.</param>
+        /// <param name="fileOrDirectoryPath">The Uri.</param>
+        /// <returns>The Subversion information.</returns>
+        /// <example>
+        /// <para>Gets Subversion information about the file or directory at <paramref name="fileOrDirectoryPath"/>.</para>
+        /// <code>
+        /// <![CDATA[
+        ///     var remoteInfo = GetRemoteInfo(new Uri("https://svn.example.com/"));
+        ///
+        ///     foreach(var svnInfoResult in remoteInfo)
+        ///     {
+        ///         Verbose("Path: {0}", svnInfoResult.Path);
+        ///     }
+        /// ]]>
+        /// </code>
+        /// </example>
+        [CakeMethodAlias]
+        [CakeAliasCategory("Info")]
+        [CakeNamespaceImport("Cake.Svn.Info")]
+        public static IEnumerable<SvnInfoResult> GetRemoteInfo(this ICakeContext context, Uri fileOrDirectoryPath)
+        {
+            return GetRemoteInfo(context, fileOrDirectoryPath, null);
+        }
+
+        /// <summary>
+        /// Gets Subversion information about the file or directory at <paramref name="fileOrDirectoryPath"/>.
+        /// </summary>
+        /// <param name="context">The Cake context.</param>
+        /// <param name="fileOrDirectoryPath">The Uri.</param>
+        /// <param name="settings">The settings.</param>
+        /// <returns>The Subversion information.</returns>
+        /// <example>
+        /// <para>Gets Subversion information about the file or directory at <paramref name="fileOrDirectoryPath"/>.</para>
+        /// <code>
+        /// <![CDATA[
+        ///     var svnInfoSettings = new SvnInfoSettings
+        ///     {
+        ///         Depth = SvnDepth.Unknown;
+        ///     };
+        ///
+        ///     var remoteInfo = GetRemoteInfo(new Uri("https://svn.example.com/"), svnInfoSettings);
```

Comment:
To avoid potential naming conflicts with other addins I suggest to name this `GetSvnRemoteInfo`